src/pages/dashboard/ui/mail/EmailEditPage.tsx

```
@@ -1,28 +1,58 @@
+    };
+    editEmail({ reservationEmailId: reservationEmailId, data: emailData,});
+  };
 
   return (
     <DashboardLayout centerContent="WOOACON 2024">
       <div className="flex flex-col gap-5 mt-8 px-7">
         <EmailInput
           type="이메일 내용 수정"
           openSelectTicket={() => setTicketModalOpen(true)}
-          allParticipantEmails={participants.map(p => p.email)}
+          allParticipantEmails={participants.map((p: { email: any; }) => p.email)}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)